### ISSUE TITLE
Admin UI issue: Missing View, Edit, and Delete language Entries 

### ISSUE THREAD

**Issue by vw98075** [jhipster/generator-jhipster#24258]
##### **Overview of the issue**

The entity view, edit, and delete language entries can't be found in the language property files. 

##### **JHipster Version(s)**

8.0.0

##### **JHipster configuration**

```
{
  "generator-jhipster": {
    "applicationType": "monolith",
    "authenticationType": "jwt",
    "baseName": "myapp",
    "buildTool": "gradle",
    "cacheProvider": "caffeine",
    "clientFramework": "vue",
    "clientTestFrameworks": [],
    "clientTheme": "spacelab",
    "clientThemeVariant": "light",
    "creationTimestamp": 1700096345577,
    "databaseType": "sql",
    "devDatabaseType": "h2Memory",
    "devServerPort": 9060,
    "enableGradleEnterprise": null,
    "enableHibernateCache": true,
    "enableSwaggerCodegen": true,
    "enableTranslation": false,
    "entities": [],
    "gradleEnterpriseHost": null,
    "jhipsterVersion": "8.0.0",
    "jwtSecretKey": "ODIzMjI0Y2EzYjAxODM4MTRjNmQ0NGE4ZTBmNTc3YzA1Njg5OTUzNDE5MmM1MjgyZjY3ZTMxODZhNjdkZWQxZmRlOTMxNGFhMDY3Y2Q3NmU3YmRjODBkN2IzMjdiOWQ1NjUyNzE1MjU2YWNhMzllOTdmZmQzM2QyNjYxNDcxYWY=",
    "messageBroker": false,
    "microfrontend": null,
    "microfrontends": [],
    "nativeLanguage": "en",
    "packageName": "com.abc.myapp",
    "prodDatabaseType": "postgresql",
    "reactive": false,
    "searchEngine": false,
    "serverPort": null,
    "serverSideOptions": ["websocket:spring-websocket", "enableSwaggerCodegen:true"],
    "serviceDiscoveryType": false,
    "testFrameworks": [],
    "websocket": "spring-websocket",
    "withAdminUi": true
  }
}
```

##### **Browsers and Operating System**

- [ ] Checking this box is mandatory (this is just to show you read everything)

**Comment by vw98075** [comment]
![Screenshot_select-area_20231115204551](https://github.com/jhipster/generator-jhipster/assets/10793038/a0282c9c-4233-4638-8fe9-15d4d1f0be54)

**Comment by qmonmert** [comment]
@vw98075 if you add `"enableTranslation": true,` in your `.yo-rc.json` file, you will have translations

cc @mshima @DanielFran

**Comment by mshima** [comment]
We did a lot of translation fixes between v8.0.0 and v8.1.0.
This issue is not reproducible.
```
% jhipster --client-framework vue --defaults --skip-install --no-enable-translation
% cat src/main/webapp/app/admin/user-management/user-management.vue | grep span
      <span id="user-management-page-heading" data-cy="userManagementPageHeading">Users</span>
          <font-awesome-icon icon="sync" :spin="isLoading"></font-awesome-icon> <span>Refresh list</span>
            <font-awesome-icon icon="plus"></font-awesome-icon> <span>Create a new user</span>
              <span>ID</span> <jhi-sort-indicator :current-order="propOrder" :reverse="reverse" :field-name="'id'"></jhi-sort-indicator>
              <span>Login</span>
              <span>Email</span>
            <th scope="col"><span>Profiles</span></th>
              <span>Created date</span>
              <span>Modified by</span>
              <span>Modified date</span>
                <span class="badge badge-info">{{ authority }}</span>
                    <span class="d-none d-md-inline">View</span>
                    <span class="d-none d-md-inline">Edit</span>
                  <span class="d-none d-md-inline">Delete</span>
% cat src/main/webapp/app/admin/user-management/user-management.vue | grep missing
```
